jquery.xblock.js

```diff
@@ -164,33 +164,62 @@
                 },
                 listenTo: function(name, callback) {
                     $this.dispatcher.bind(name, callback);
+                },
+                children: function(element) {
+                    return $(element).data('xblock-children');
+                },
+                childMap: function(block, childName) {
```

Comment:
@bradenmacdonald nit: confusing name - I was expecting it to build a map of children, not return children by name. `getChildbyName` maybe?